scipy/stats/stats.py

```diff
@@ -3565,6 +3568,159 @@ def count_rank_tie(ranks):
     # Limit range to fix computational errors
     return KendalltauResult(min(1., max(-1., tau)), pvalue)
 
+WeightedTauResult = namedtuple('WeightedTauResult', ('correlation', 'pvalue'))
+
+
+def weightedtau(x, y, rank=True, weigher=lambda x: 1./(x+1), additive=True):
+    """
+    Computes a weighted version of Kendall's tau.
+
+    The weighted tau is a weighted version of Kendall's tau in which
+    exchanges of high weight are more influent than exchanges of low
+    weight. The default parameters compute the additive hyperbolic version
+    of the index, tau_h, which has been shown to provide the best balance
```

Comment:
You want to surround ``tau_h`` by double backticks, so it gets rendered as code. Otherwise the ``_`` will lead to strange rendering issues.